src/app/data-fetchers/country-list.js

```diff
@@ -42,7 +42,7 @@ class CountryList {
   }
 
   onUpdate (listener: Callback<Array<Country>>) {
-    this._listeners.subscribe(listener)
+    this._publisher.subscribe(listener)
```

Comment:
`this._publisher.subscribe` reads unnatural.
feels like it's the publisher is subscribing to something
```suggestion
    this._publisher.addSubscription(listener)
```

```suggestion
    this._publisher.addListener(listener)
```

Reply:
Renamed to `.addSubscriber` and `.removeSubscriber`, since subscriber is used most commonly :)